ui/src/main/java/edu/wpi/grip/ui/codegeneration/CodeGenerationOptions.java

```diff
@@ -0,0 +1,128 @@
+package edu.wpi.grip.ui.codegeneration;
+
+import edu.wpi.grip.core.settings.ProjectSettings;
+
+import java.io.File;
+import java.util.HashMap;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Holds options for code generation.
+ */
+public class CodeGenerationOptions extends HashMap<String, Object> {
```

Comment:
Cool. But I'm probably going to make this a bean-style class. I made it a map to begin with because I though there'd only be Java or C++ or Python options. But now it's just going to have all of them, so no need